### ISSUE TITLE
In the SLA report an planned downtime is listed under the last/worst outages but it's not calculated into the graphs.

### ISSUE THREAD

**Issue by Menthe11** [sni/Thruk#441]
In the SLA report an planned downtime is listed under the last/worst outages but it's not calculated into the graphs.

SLA Report:
06 Dez 2014  09:44:17 - 06 Dez 2014  18:46:51        (0d 9h 2m 34s)
PING CRITICAL - Packet loss = 100%

EventLog Nagios/Thruk:
[2014-12-06
09:38:47] HOST DOWNTIME ALERT: $HOSTNAME$;STARTED; Host has entered a 
period of scheduled downtime

[2014-12-06
09:38:47] EXTERNAL COMMAND: 
SCHEDULE_HOST_DOWNTIME;$HOSTNAME$;1417855119;1417892400;1;0;7200;nagiosadmin;umbauarbeiten

[2014-12-06
09:37:59] SERVICE DOWNTIME ALERT: $HOSTNAME$;PING;STARTED; Service has 
entered a period of scheduled downtime

[2014-12-06
09:37:59] EXTERNAL COMMAND: 
SCHEDULE_HOST_SVC_DOWNTIME;$HOSTNAME$;1417855069;1417892400;1;0;7200;nagiosadmin;umbauarbeiten

[2014-12-06
20:00:00] SERVICE DOWNTIME ALERT: $HOSTNAME$;PING;STOPPED; Service has exited
 from a period of scheduled downtime

[2014-12-06
20:00:00] HOST DOWNTIME ALERT: $HOSTNAME$;STOPPED; Host has exited from a 
period of scheduled downtime
